### DISCUSSION TITLE
Timeline zoom for the editor's bottom timeline

### DISCUSSION THREAD

**Discussion by minetoblend** [ppy/osu#18748] [Ideas]
For long maps the bottom timeline doesn't really work well for navigating anymore since everything's squished together so much that a few pixels off might already send you somewhere 30 seconds from where you actually wanna go.
I think a nice way to handle that interaction-wise would  be to shift-click drag an area to zoom in on, and shift click again to zoom back out.
Especially for collab maps, where you're only gonna wanna navigate around your own part anyways, this would be a big help.

**Comment by peppy** [comment]
Hmm, this is an interesting one. The bottom timeline was always supposed to be an overview of the full map, so I'm not sure how I feel about zooming. If we did add any kind of zoom, I think it would be a temporary operation, like dragging upwards during a seek to get more precise control (I've seen this done elsewhere and it works quite well).
